catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/utils/BrickVariableSpinnerDataInteractionWrapper.java

```diff
@@ -89,6 +88,14 @@ private static void enterTextOnDialogue(int dialogueId, String textToEnter) {
 				.perform(click());
 	}
 
+	public BrickVariableSpinnerDataInteractionWrapper performSelectList(String selection) {
```

Comment:
either go with BrickSpinnerDataInteractionWrapper for that particular check / case, or maybe refactor this wrapper to extend BrickVariableSpinnerDataInteractionWrapper from BrickSpinnerDataInteractionWrapper to not have the code duplication.